pkgs/development/tools/build-managers/gprbuild/gpr-project-darwin-rpath-hook.sh

```diff
@@ -0,0 +1,10 @@
+fixDarwinRpath() {
+    for f in $(find $out -type f -executable); do
+        install_name_tool -id $f $f || true
+        for rpath in $(otool -L $f | grep rpath | awk '{print $1}'); do
+            install_name_tool -change $rpath $out/lib/$(basename $rpath) $f || true
```

Comment:
It' probably sensible to use `${!outputLib}` instead of `$out` here in case the package we are building has a separate lib output.